src/main/java/org/primeframework/mvc/action/config/ActionConfiguration.java

```diff
@@ -60,6 +60,8 @@ public class ActionConfiguration {
 
   public final Map<HTTPMethod, ExecuteMethodConfiguration> executeMethods;
 
+  public final List<String> fallbackActionMessageURIs;
```

Comment:
This is "internal" to prime, so not a big deal, but if the annotation is `AlternateMessageResources` should this be `alternateMessageURIs`? 

Naming consistency can help find the code in the future.

Reply:
Fixed